src/SUMMARY.md

```diff
@@ -35,6 +35,8 @@
     - [Inferred Types](./variables/inferred_types.md)
     - [Challenges](./variables/challenges.md)
 
+- [Benefits of java](./benefits_of_java.md)
```

Comment:
FYI, I think that showing the benefits of Java is a good idea, especially at the present time as some new technologies have become increasingly dominant in the ideas of developers and organizations, so the importance of Java has become invisible in the required way, so the public and the private should be reminded of its importance and benefits. 

Also, to further encourage the reader and prepare him to continue reading and liking the book